src/ui/molecules/footer/footer.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../styles/themes.scss';
+
+.okp4-footer-main {
+  @include with-theme() {
+    background: themed(footer-background);
+    width: calc(100% - 80px);
+    min-height: 40px;
+    padding: 45px 20px 45px 60px;
+    align-items: center;
+    justify-content: space-around;
```

Comment:
If you don't mind, I'd prefer use `space-between` property instead to better fit with UX wireframes and will revert to `space-around` in media query

Reply:
Sure thing I'm on it! 😄